l10n/en/footer-refresh.ftl

```diff
@@ -40,3 +40,5 @@ footer-refresh-websites-legal = Legal
 footer-refresh-community-participation-guidelines = Community Participation Guidelines
 footer-refresh-about-this-site = About this site
 footer-refresh-language = Language
+footer-refresh-primary-nav-aria-label = Footer primary
+footer-refresh-secondary-nav-aria-label = Footer secondary
```

Comment:
Maybe to start a discussion before this goes to l10n — is there a more helpful distinction between these two sections, to capture that in the labels right away?

I see the two e.g. as:

```suggestion
footer-refresh-primary-nav-aria-label = Footer links and external accounts
footer-refresh-secondary-nav-aria-label = Footer legal info
```

Reply:
I think what we have now is generic enough not to pen things into certain categories. The secondary links contain more than just legal links for example (Community Participation Guidelines, About this site etc)